studio/src/rte/rte-field.js

```diff
@@ -161,6 +162,64 @@ class MnemonicNodeView {
     }
 }
 
+class DividerNodeView {
```

Comment:
using `sp-divider` will by default force you to have spectrum to render it, unless it is downgraded to `hr` during hydration.

I would enable OOTB  hr feature (https://prosemirror.net/docs/ref/#schema-basic.nodes.horizontal_rule) and based on consonant/spectrum flags, augment it to `sp-divider` if needed.

Reply:
done. now only using <hr>